src/ServiceProviders/CIProviders/GitLabCI/GitLabCIProvider.php

```diff
@@ -158,6 +158,10 @@ protected function setGitLabCIEnvironmentVars(CIState $ci_env)
             $data = ['key' => $key, 'value' => $value];
             $this->gitlabCIAPI($data, $gitlab_url);
         }
+
+        // Also set the GitLab URL for future use.
+        $data = ['key' => 'TERMINUS_BUILD_TOOLS_PROVIDER_GIT_GITLAB_URL', 'value' => $this->getGITLABURL()];
+        $this->gitlabCIAPI($data, $gitlab_url);
```

Comment:
You need the GitLab URL to retrieve this value, which contains the GitLab URL, right? Seems this isn't helpful, unless I'm misunderstanding something.

Reply:
This is particularly needed when trying to retrieve PR information for GitLab repos. When run via user (on their machine), the value comes from the Terminus config file. However, in the Build Tools docker container, that value isn't set (because you have no config file). Which means requests to the GitLab API go to gitlab.com instead of your self-hosted instance. By setting this environment variable here, it gets injected automatically into every CI build and BT picks it up as the config option.

Reply:
Ah, right, you don't need to ask the API for the value because it will be re-applied as an environment variable when your tests run. Obvious enough, but could you add one more line to the comment about that anyway?

Reply:
Done.